cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -65,12 +65,14 @@ mode and use a `cloud-init` disk, the virtual machine will lose its network
 connectivity after being restarted. As a workaround, remove the `HWADDR` line
 in the file `/etc/sysconfig/network-scripts/ifcfg-eth0`.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1708680[*BZ#1708680*])
-+
-[NOTE]
-====
-Avoid this issue by using the recommended `masquerade` binding method for
-the default Pod network.
-====
+
+* Masquerade does not currently work with CNV. Due to an upstream issue,
+you cannot connect a virtual machine to the default Pod network while in Masquerade mode.
+
+* Creating a NIC with Masquerade in the wizard doesn't allow you to specify the `port` option.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1725848[*BZ#1725848*])
+
+* RWX (Read, Write, Execute) file systems are the only supported storage for Live Migration, UI, and importing VMWare virtual machines.
```

Comment:
Same question about the BZ link.

Reply:
This Release Note came directly from QE feedback. I will try to find what bug it is associated with.